actors/builtin/market/market_state.go

```diff
@@ -22,6 +22,12 @@ const epochUndefined = abi.ChainEpoch(-1)
 // activate deal (miner)
 // end deal (miner terminate, expire(no activation))
 
+type role int
+const (
+	client role = iota
+	provider
```

Comment:
`CAPITALIZE` these please, they're otherwise confusing to see in the code (where client or provider usually refers to some address)

Reply:
This is done.